go-controller/pkg/ovn/egressip.go

```diff
@@ -2222,6 +2237,28 @@ func (e *egressIPDial) dial(ip net.IP, timeout time.Duration) bool {
 	return true
 }
 
+func (oc *Controller) isReachableViaGRPC(node *egressNode, health_check_port int) bool {
+	// Check if we need to do node reachability check
+	if oc.eIPC.egressIPTotalTimeout == 0 {
```

Comment:
I'm just wondering ...
given that this and len(node.mgmtIPs) are common to isReachableViaDial and isReachableViaGRPC ... should we move this into isReachable?
No strong opinion and only if it improves readability. Otherwise, I don't care about this little code duplication.

Reply:
Definitely! I opted for duplicating that to leave isReachableViaDial undisturbed. But you are right in that this is a bit of duplication.

I will make that change when I do the missing work for this PR: unit tests and TLS.